requirements.in

```diff
@@ -3,7 +3,7 @@
 #    python -m scripts.install_third_party
 #
 # to update `requirements.txt` and `third_party/python_libs` accordingly.
-apache-beam[gcp]==2.31.0
+apache-beam[gcp]==2.33.0rc1
```

Comment:
Should we move to the last stable release (32) instead? Seems like 33 isn't quite ready yet.

Reply:
2.32.0 sadly doesn't support typing_extensions 3.10.0.2

Reply:
Ack'd, sounds good to me.

Reply:
tbh I think we can keep it and reevaluate it before our October release.